modules/simple_bucket/variables.tf

```diff
@@ -78,6 +78,17 @@ variable "retention_policy" {
   default = null
 }
 
+variable "cors" {
+  description = "Configuration of CORS for bucket with structure as defined in https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/storage_bucket#cors."
+  type = list(object({
```

Comment:
we should use `any` and look up. This way users won't have to provide empty values for unused fields. [Example](https://github.com/terraform-google-modules/terraform-google-cloud-storage/blob/a0982c94f92df37d903af0a85e7ee5fe1e655d1e/variables.tf#L192)